cirq-rigetti/cirq_rigetti/service.py

```diff
@@ -112,7 +112,9 @@ def list_quantum_processors(client: Optional[QCSClient]) -> List[str]:  # pragma
         return list_quantum_processors(client=client)
 
     @staticmethod
-    def get_quilt_calibrations(quantum_processor_id: str, client: Optional[QCSClient]) -> str: # pragma: no cover
+    def get_quilt_calibrations(
+        quantum_processor_id: str, client: Optional[QCSClient]
+    ) -> str:  # pragma: no cover
```

Comment:
This likely doesn't need to be tested as it's just a convenience method for an API call, and there's no mock for `QCSClient`.